src/db/database.cr

```diff
@@ -4,21 +4,20 @@ require "weak_ref"
 module DB
   # Acts as an entry point for database access.
   # Connections are managed by a pool.
-  # The connection pool can be configured from URI parameters:
+  #
+  # ## Database URI
+  #
+  # Connection parameters are configured in a URI. The format is specified by the individual database drivers. See [the book](https://github.com/crystal-lang/crystal-book/tree/master/database#open-database) for examples.
```

Comment:
I am not sure about linking the book as samples, there are other samples that assume specific driver is used. 
Yet, if a link is to be used, use https://crystal-lang.org/docs/ instead of the source